src/chapters/angular-lsn3/studio.rst

```diff
@@ -0,0 +1,200 @@
+Studio: Angular, Part 3
+========================
+
+In this studio, you must combine data-binding and attribute directives to
+dynamically change the appearance or behavior of HTML elements.
+
+Getting Started
+----------------
+
+This studio uses the same mission planner repository as Angular studio parts 1
+and 2.
+
+#. Open the `mission planner repository <https://github.com/LaunchCodeEducation/angular-lc101-mission-planner/tree/studio-3>`__
+   in VSCode.
+#. Run ``git status`` to see if you have any uncommitted work, if you do
+   resolve it.
+#. Checkout the ``studio-3`` branch.
+#. Run ``npm install`` to download dependencies.
+#. Run ``ng serve`` to build and serve the project.
+
+When viewed in your browser, the project should look like this:
+
+.. figure:: figures/angular-lsn3-starter-page.png
+   :alt: Angular studio 3 starting point.
+
+*Mission Name* and *Carrier Rocket* are still editable, but the functionality
+for the crew, equipment, and experiments have been removed.
+
+Part 1: Select Cargo
```

Comment:
i didnt catch this earlier but part 1 does not include instructions on maxItem handling, even though that's included in the solution and the gifs on this page

Reply:
Oops.  Fixed.